src/platforms/apple/common/migration.mdx

```diff
@@ -24,7 +24,11 @@ application.
 
 - Use a BOOL in SentryOptions instead of NSNumber to store booleans.
 
-- We removed [enabled](https://github.com/getsentry/sentry-cocoa/blob/5.2.2/Sources/Sentry/include/SentryOptions.h#L63) on the SentryOptions.
+- We removed [enabled](https://github.com/getsentry/sentry-cocoa/blob/5.2.2/Sources/Sentry/include/SentryOptions.h#L63) on the SentryOptions,
+but we brought them back in 6.0.7 with slightly changed functionality. Previously setting
+an empty or wrong DSN also set `enabled` to `false`. This side effect is removed now. Setting
+the DSN has no impact on `enabled`. If the DSN is nil or empty or `enabled` is set
+to `false`, the client doesn't send anything to Sentry.
```

Comment:
```suggestion
to `false`, the client won't send any event to Sentry.
```

Reply:
But it's not just about events but also, envelopes, sessions, and user feedback for example

Reply:
got your point, maybe `any data`? also happy to keep as it is

Reply:
Any data makes sense, yes.